parity/main.rs

```diff
@@ -85,6 +85,10 @@ Options:
   --jsonrpc-url URL        Specify URL for JSON-RPC API server [default: 127.0.0.1:8545].
   --jsonrpc-cors URL       Specify CORS header for JSON-RPC API responses [default: null].
 
+  --author ADDRESS         Specify the block author (aka "coinbase") address for sending block rewards
+                           from sealed blocks [default: 0037a6b811ffeb6e072da21179d11b1406371c63].
```

Comment:
is this a common practice, use some address unaccessible for user as a default one?